extension/src/config/config.js

```diff
@@ -65,6 +67,10 @@ function loadAndValidateConfig() {
       'Adyen integration configuration is not provided in the JSON format'
     )
   }
+
+  if (_.isEmpty(config.apiExtensionBaseUrl))
```

Comment:
yes but only needed in `ensureResources` and it's not required 🤔

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/542/commits/da3b72a09044a0239b1fbdcd4d88f488b0aa14a7